reference-adapter/mgramseva-ifix-adapter/src/main/java/org/egov/ifix/utils/EventConstants.java

```diff
@@ -87,6 +87,7 @@ public class EventConstants {
     public static final String USER_TYPE = "userType";
     public static final String CLIENT_SECRET = "client_secret";
     public static final String CLIENT_ID = "client_id";
-    public static final String PSPCL_VENDOR_NAME = "PSPCL";
+    public static final String PSPCL_VENDOR_NAME = "TATA";
```

Comment:
Keep this as an env var.
If already done, remove this line.

Reply:
Removed it